src/platform/monitoring/DowntimeNotification/util/helpers.js

```diff
@@ -26,6 +27,29 @@ export function getStatusForTimeframe(startTime, endTime) {
   return externalServiceStatus.ok;
 }
 
+export function createGlobalMaintenanceWindow({
+  startTime,
+  endTime,
+  externalServices = defaultExternalServices,
+}) {
+  return [
+    {
+      attributes: {
+        externalService: 'global',
```

Comment:
_most_ of our forms do not designate the services they depend on in the downtime config. decided to add a "global" value for those. Later on I'll point out how this gets piped through the config. 

this function generates a `/maintenance_windows` response data as if each service is down for the fallback window.

Reply:
Any reason we have `global` as a floating string here instead of the constant?

Reply:
mostly just me getting it to work and quickly moving on. i'll update it.